lib/task-data/tasks/install-esx.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         users: [],
         networkDevices: [],
         dnsServers: [],
+        ntpServers: [],
```

Comment:
After the task-schema upfront validation is enabled, this line should not put here. see this PR: https://github.com/RackHD/on-tasks/pull/257